bezier-rs/docs/interactive-docs/src/components/BezierDrawing.ts

```diff
@@ -5,6 +5,14 @@ import { BezierCallback, BezierPoint, BezierStyleConfig, Point, WasmBezierMutato
 // Offset to increase selectable range, used to make points easier to grab
 const FUDGE_FACTOR = 3;
 
+const MAP_LENGTH_TO_MUTATOR_INDEX: {
```

Comment:
Based on its name and looking at it, I really don't have any idea what this does or means. Let's chat about that and pick a name that hopefully can convey more understanding.

Reply:
I refactor this logic a little and added a more descriptive comment. Does it help in understanding the purpose of the constant?